src/routes/auth/auth.service.ts

```diff
@@ -58,6 +68,26 @@ export const authService = {
 
     return { ...dbResponse, token }
   },
+  // TODO: zod for token & email?
+  async verifyEmail(email: string, token: string): Promise<void> {
+    const payload = await verify(token, process.env.JWT_SECRET!)
```

Comment:
WDYT about using an auth middleware to verify the JWT and set the payload in the context? For example:
https://github.com/SlavaMelanko/portal-back-v2/blob/main/src/middleware/auth.ts